package.json

```diff
@@ -105,7 +105,7 @@
     "uniqid": "5.4.0",
     "url-parse": "1.5.10",
     "webpack": "5.76.0",
-    "webpack-cli": "4.9.1",
+    "webpack-cli": "4.10.0",
```

Comment:
Had to update this package due to the error `[webpack-cli] TypeError: cli.isMultipleCompiler is not a function` when I was trying to run `npm start`.
Google told me I need to update it and it worked: https://github.com/webpack/webpack/issues/15951